python/paddle/fluid/tests/book/word2vec/no_test_word2vec_new_api.py

```diff
@@ -100,23 +100,25 @@ def event_handler(event):
                 word_dict, N))
 
             if avg_cost < 5.0:
-                trainer.params.save(save_path)
+                trainer.save_params(save_path)
```

Comment:
Not only `save_params` should be supported in the trainer, but also `save_checkpoint` should be supported by the trainer to restore training progress.

Reply:
Sure, thank you for pointing out!